doc/README.twom.md

```diff
@@ -0,0 +1,189 @@
+This document describes the twom database format for key-value store within Cyrus.
+
+## Skiplists
+
+A [skiplist](https://en.wikipedia.org/wiki/Skip_list) is a data structure gives O(log N) access to data; by creating multiple linked lists.  Every node contains a pointer to the immediately next node in a sorted order, however some nodes also contain links that “skip over” some number of nodes, hence “skip”.  I won’t describe in detail how skiplists work here, as that’s described well in other places.
```

Comment:
In addition to the Wikipedia, this paper gives a good overview https://arxiv.org/abs/2403.04582

Reply:
Nice :)  I think the wikipedia link is enough to have in this doc, but maybe I should read it!